cms/envs/aws.py

```diff
@@ -303,6 +303,19 @@
 ##### X-Frame-Options response header settings #####
 X_FRAME_OPTIONS = ENV_TOKENS.get('X_FRAME_OPTIONS', X_FRAME_OPTIONS)
 
+##### OAUTH2 Provider ##############
+if FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
```

Comment:
Nitpick: is there a reason these options are all defined inside this block? Having them set shouldn't make any difference if the feature flag isn't turned on, but if something later on in the configuration-parsing process decides to flip this flag on, then these config values would already be defined, and things would just work. By contrast, the way the code is right now, you would have to re-execute all this logic to turn on the feature, in addition to flipping the feature flag on.

Realistically, I'm not sure if we ever would get into this situation, which is why this is just a nitpick. But I wanted to mention it, just in case.